test/directives/vector_directive3.f90

```diff
@@ -1,15 +1,22 @@
-! RUN: %flang -c %s 2>&1 | FileCheck %s -allow-empty --check-prefix=CHECK
+!! check for pragma support for !dir$ vector always
+!RUN: %flang -S -O2 -emit-llvm %s -o - | FileCheck %s
+!CHECK: define void @sumsimd_{{.*$}}
+!CHECK: {{.*}}!llvm.access.group{{.*}}
+!CHECK: vector.ph:{{.*}}
+!CHECK: vector.body:{{.*}}
+!CHECK: {{.*}}shufflevector{{.*}}
+!CHECK: {{.*}}add <2 x i64>{{.*}}
+!CHECK: {{.*}}"llvm.loop.parallel_accesses"{{.*}}
```

Comment:
Should check that the access mentioned in the parallel_accesses matches the access.group above.

Reply:
Done.